src/main/java/org/opentripplanner/standalone/config/NodeAdapter.java

```diff
@@ -84,6 +84,10 @@ JsonNode asRawNode(String paramName) {
         return param(paramName);
     }
 
+    public JsonNode asRawNode() {
+        return json;
+    }
+
```

Comment:
Use the  #asRawNode() method instead of breaking encapsulation - this could lead to false-positive warnings when starting the server...

Reply:
I removed this method, as it is not needed anymore.